opennms-base-assembly/src/main/filtered/etc/events/opennms.pollerd.events.xml

```diff
@@ -618,4 +618,17 @@
         </logmsg>
         <severity>Normal</severity>
     </event>
+    <event>
+        <uei>uei.opennms.org/internal/poller/servicePollStatus</uei>
+        <event-label>OpenNMS-defined poller event: servicePollStatus</event-label>
+        <descr>
+            &lt;p&gt;Service Polling Status Event for the %service% service
+            on interface %interface% as covered by a poller package, and
+            managed.
+        </descr>
+        <logmsg dest='logndisplay'>
```

Comment:
We think this should be made 'dbonly' be default in order to avoid spamming the event log in the Web UI when this is enabled.